definitions/npm/react-redux_v5.x.x/flow_v0.68.0-/react-redux_v5.x.x.js

```diff
@@ -0,0 +1,212 @@
+declare module "react-redux" {
+  import type { ComponentType, ElementConfig } from 'react';
+
+  // These types are copied directly from the redux libdef. Importing them in
+  // this libdef causes a loss in type coverage.
+  declare type DispatchAPI<A> = (action: A) => A;
+  declare type Dispatch<A: { type: $Subtype<string> }> = DispatchAPI<A>;
+  declare type Reducer<S, A> = (state: S | void, action: A) => S;
+  declare type Store<S, A, D = Dispatch<A>> = {
+    dispatch: D;
+    getState(): S;
+    subscribe(listener: () => void): () => void;
+    replaceReducer(nextReducer: Reducer<S, A>): void
+  };
+
+  declare export class Provider<S, A, D> extends React$Component<{
+    store: Store<S, A, D>,
```

Comment:
This is the change: the 3rd parameter `D` has been added, so that D can be overriden instead of being the default `<Dispatch<A>>`